### DISCUSSION TITLE
Good resources for learning git or github?

### DISCUSSION THREAD

**Discussion by jtgorman** [uillinois-community/uillinois-community.github.io#20] [Q&A] [answered]
I remember quite liking the interactive tutorial/game [githug](https://github.com/Gazler/githug). And the [github docs](https://docs.github.com/en) are pretty good too for github-specific info.

Are there other good resources for learning either git or github?

**Comment by areynold** [comment, marked as answer]
I used to point to [Atlassian's docs](https://www.atlassian.com/git/tutorials) pretty regularly, especially the [Comparing Workflows](https://www.atlassian.com/git/tutorials/comparing-workflows) page in the Collaborating section. That's been a significant hurdle for a lot of folks when they try to move from basic repo operations to working with a team.

There was another page, that I wish I could find again, written for Subversion users. In addition to covering the same-name, different-effect commands, it talked about some of the process decisions that need to be agreed on when moving from svn to git.

**Comment by edthedev** [comment]
A couple more resources:

- [Illinois Media Space Videos with the keyword Git](https://mediaspace.illinois.edu/esearch/search?keyword=git)
- [Software Carpentry Lessons for a Git Novice](https://swcarpentry.github.io/git-novice/)
